go.mod

```diff
@@ -5,46 +5,44 @@ require (
 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b // indirect
 	github.com/socketplane/libovsdb v0.0.0-20170116174820-4de3618546de
-	github.com/vishvananda/netlink v0.0.0-20181108222139-023a6dafdcdf
-	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37 // indirect
-	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
-	k8s.io/apimachinery v0.18.3
+	github.com/vishvananda/netlink v1.1.0
+	k8s.io/apimachinery v0.19.1
 	k8s.io/client-go v0.0.0-00010101000000-000000000000
 	kubevirt.io/qe-tools v0.1.6
 )
 
-// Pinned to kubernetes-1.18.3
+// Pinned to kubernetes-1.19.1
 replace (
-	k8s.io/api => k8s.io/api v0.18.3
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.3
-	k8s.io/apimachinery => k8s.io/apimachinery v0.18.3
-	k8s.io/apiserver => k8s.io/apiserver v0.18.3
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.3
-	k8s.io/client-go => k8s.io/client-go v0.18.3
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.18.3
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.18.3
-	k8s.io/code-generator => k8s.io/code-generator v0.18.3
-	k8s.io/component-base => k8s.io/component-base v0.18.3
-	k8s.io/cri-api => k8s.io/cri-api v0.18.3
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.3
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.3
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.18.3
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.18.3
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.3
-	k8s.io/kubectl => k8s.io/kubectl v0.18.3
-	k8s.io/kubelet => k8s.io/kubelet v0.18.3
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.18.3
-	k8s.io/metrics => k8s.io/metrics v0.18.3
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.18.3
+	golang.org/x/text => golang.org/x/text v0.3.3
+	k8s.io/api => k8s.io/api v0.19.1
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.1
+	k8s.io/apimachinery => k8s.io/apimachinery v0.19.1
+	k8s.io/apiserver => k8s.io/apiserver v0.19.1
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.1
+	k8s.io/client-go => k8s.io/client-go v0.19.1
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.1
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.1
+	k8s.io/code-generator => k8s.io/code-generator v0.19.1
+	k8s.io/component-base => k8s.io/component-base v0.19.1
+	k8s.io/cri-api => k8s.io/cri-api v0.19.1
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.19.1
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.1
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.1
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.1
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.1
+	k8s.io/kubectl => k8s.io/kubectl v0.19.1
+	k8s.io/kubelet => k8s.io/kubelet v0.19.1
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.1
+	k8s.io/metrics => k8s.io/metrics v0.19.1
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.19.1
 )
 
-go 1.13
+go 1.14
```

Comment:
I'm afraid we have to stay on Go 1.13 for a while. Could you keep it there?

Reply:
reverted